src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.model;
+
+import java.math.BigDecimal;
+
+public class Fruit {
+    private String name;
+    private BigDecimal quantity;
```

Comment:
quantity is not a fruit property. quantity is Storage responsibility.
You can delete this class